src/pages/home/report/EmojiPickerMenu/index.native.js

```diff
@@ -1,14 +1,19 @@
 import React, {Component} from 'react';
 import {View, FlatList, Text} from 'react-native';
 import PropTypes from 'prop-types';
+import compose from '../../../../libs/compose';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../../../../components/withWindowDimensions';
 import CONST from '../../../../CONST';
 import styles from '../../../../styles/styles';
 import emojis from '../../../../../assets/emojis';
 import EmojiPickerMenuItem from '../EmojiPickerMenuItem';
+import dynamicEmojiSize from './dynamicEmojiSize';
 
 const propTypes = {
     /** Function to add the selected emoji to the main compose text input */
     onEmojiSelected: PropTypes.func.isRequired,
+
+    ...windowDimensionsPropTypes,
```

Comment:
NAB: I see there are many places in our code where this is prop is not documented but as per our style guide it would be nice if you include a JS comment doc for this for consistency.